tracing-undertow/src/main/java/com/palantir/tracing/undertow/TracedOperationHandler.java

```diff
@@ -25,18 +25,19 @@
 import io.undertow.util.AttachmentKey;
 
 /**
- * Extracts Zipkin-style trace information from the given HTTP request and sets up a corresponding {@link
- * com.palantir.tracing.Trace} and {@link com.palantir.tracing.api.Span} for delegating to the configured {@link
- * #delegate} handler. See <a href="https://github.com/openzipkin/b3-propagation">b3-propagation</a>.
+ * Extracts Zipkin-style trace information from the given HTTP request and sets up a corresponding
+ * {@link com.palantir.tracing.Trace} and {@link com.palantir.tracing.api.Span} for delegating to the configured
```

Comment:
I really appreciate the change to join javadoc tags on a single line where possible, thanks @dansanduleac!